llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -1410,7 +1410,7 @@ class VPIRInstruction : public VPRecipeBase {
   InstructionCost computeCost(ElementCount VF,
                               VPCostContext &Ctx) const override;
 
-  Instruction &getInstruction() { return I; }
+  Instruction &getInstruction() const { return I; }
```

Comment:
Independent?

Reply:
Done in 466ff3ed70e50343c9617de318e1b40efa03529d, thanks!